config/paths.js

```diff
@@ -34,7 +34,9 @@ function resolveApp(relativePath) {
 // config after eject: we're in ./config/
 module.exports = {
   appBuild: resolveApp('build'),
+  appFavico: resolveApp('src/favicon.ico'),
```

Comment:
Favicon is not required anymore. We can remove it from the list of files to check.